src/runtime/math.py

```diff
@@ -709,3 +709,43 @@ def remainder(x: f64, y: f64) -> f64:
     if x - y*f64(q) > y*f64(q + i64(1)) - x:
         return x - y*f64(q + i64(1))
     return x - y*f64(q)
+
+
+@overload
+def frexp(x:f64) -> tuple[f64,i16]:
```

Comment:
You need not change it here, but I am just curious to know why you used `i16` to store the exponent value in case of `f64` and `i8` to store the exponent value in case of `f32`.

Reply:
I tried to get the most accurate size needed for the exponent, as for `f32` it would need 8 bits in the worst case scenario and for `f64` it would need less than 16 bits that's why I used `i8` and `i16`.

Reply:
Ok, good, thanks!